.env.example

```diff
@@ -2,29 +2,17 @@
 
 RELAYER_NEUTRON_CHAIN_RPC_ADDR=tcp://host.docker.internal:16657
 RELAYER_NEUTRON_CHAIN_REST_ADDR=http://host.docker.internal:1316
-RELAYER_NEUTRON_CHAIN_CHAIN_ID=test-1
 RELAYER_NEUTRON_CHAIN_HOME_DIR=/data/test-1
 RELAYER_NEUTRON_CHAIN_SIGN_KEY_NAME=demowallet1
-RELAYER_NEUTRON_CHAIN_TIMEOUT=1000s
 RELAYER_NEUTRON_CHAIN_GAS_PRICES=0.5stake
 RELAYER_NEUTRON_CHAIN_GAS_LIMIT=10000000
 RELAYER_NEUTRON_CHAIN_GAS_ADJUSTMENT=2.0
 RELAYER_NEUTRON_CHAIN_CONNECTION_ID=connection-0
-RELAYER_NEUTRON_CHAIN_CLIENT_ID=07-tendermint-0
-RELAYER_NEUTRON_CHAIN_DEBUG=true
 RELAYER_NEUTRON_CHAIN_KEYRING_BACKEND=test
-RELAYER_NEUTRON_CHAIN_OUTPUT_FORMAT=json
-RELAYER_NEUTRON_CHAIN_SIGN_MODE_STR=direct
 
 RELAYER_TARGET_CHAIN_RPC_ADDR=tcp://host.docker.internal:26657
-RELAYER_TARGET_CHAIN_CHAIN_ID=test-2
 RELAYER_TARGET_CHAIN_ACCOUNT_PREFIX=neutron
 RELAYER_TARGET_CHAIN_VALIDATOR_ACCOUNT_PREFIX=neutrondeveloper
-RELAYER_TARGET_CHAIN_TIMEOUT=1000s
-RELAYER_TARGET_CHAIN_CONNECTION_ID=connection-0
-RELAYER_TARGET_CHAIN_CLIENT_ID=07-tendermint-0
-RELAYER_TARGET_CHAIN_DEBUG=true
-RELAYER_TARGET_CHAIN_OUTPUT_FORMAT=json
```

Comment:
I understand why you removed chain ID, client ID and connection ID, but what's the point about the other fields? I can see you introduced default values for these parameters, but why aren't they in the example config files anymore?

Reply:
these values are:

1. not necessary in `.env.example.dev`
2. necessary in `.env.example`, please restore them